tests/unit/components/ont/OntTubeSelectedList.spec.js

```diff
@@ -58,7 +58,7 @@ describe('OntTubeSelectedList', () => {
       expect(wrapper.find('tbody').findAll('td')[3].text()).toEqual('basecalls')
       expect(wrapper.find('tbody').findAll('td')[4].text()).toEqual('ONT_PromethIon')
       expect(wrapper.find('tbody').findAll('td')[5].text()).toEqual('1')
-      expect(wrapper.find('tbody').findAll('td')[6].text()).toEqual('Remove')
+      expect(wrapper.find('tbody').findAll('td')[6].text()).toEqual('Add   Remove')
     })
 
```

Comment:
There is no test case to cover tube selection when the add button is clicked.

Reply:
will look into the test case. Thanks!

Reply:
added test cases for the button.